transport/src/main/java/io/netty/bootstrap/AbstractBootstrap.java

```diff
@@ -218,6 +220,12 @@ public B validate() {
         return (B) this;
     }
 
+    private void ensureChannelFactory() {
+        if (channelFactory == null) {
+            channelFactory = ChannelFactoriesRegistry.getFactoryForEventLoopGroup(group.getClass(), isServerSide());
```

Comment:
@rkapsi @fenik17  getDefaultChannelFactory() by default will return null as it will return for type of EventLoopGroups which are not pre-registered.
I will update the patch soon